Purchases/Misc/Deprecations.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  Deprecations.swift
+//
+//  Created by Nacho Soto on 11/15/21.
+
+import StoreKit
+
+// swiftlint:disable unused_setter_value
+
+extension CustomerInfo {
```

Comment:
nitpick: let's use `public extension` and then remove the visibility from the method, for consistency with the `public extension Purchases` below